01-js-intro-and-jquery/keypoints/1_7,6,8,9,10/resources/script/script.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by MaximilianoDaniel on 10/09/2015.
+ */
+
+
+
+var onPageReady = function () {
+    var input = $('.alias');
+    var button = $('#btn-hit-me');
+    var section = $('section.hidden');
+
+    section.fadeIn('normal', function () {
+        input.focus();
+    });
+
+
+
+    button.click(function () {
+        var inputValue = input.val() || 'guest';
+
+        $.ajax({
+            url: 'http://bootcamp.aws.af.cm/welcome/' + inputValue,
+            success: function (result) {
+                section.text(result.response);
+            },
+            error: function (a, b, c) {
```

Comment:
It's ok for this one but try to use a more detailed parameters names, what is a? what is b? what is c? For the next time use other parameters names, for this one it's OK. No need to change